### PR TITLE
Use external PRRTE 3.1.5 in OpenMPI 5.0.3

### PR DESCRIPTION
**Pull request by bartoldeman**
(created using `eb --new-pr`)

needs https://github.com/easybuilders/easybuild-easyblocks/pull/3347

### CODE REVIEW

**Comment by bartoldeman** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--include-easyblocks-from-pr 3347"

**Comment by bedroge** [comment]
Test report by @bedroge
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3347
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.12
See https://gist.github.com/bedroge/85ad0fa185520b0b25153601d8909382 for a full test report.

File: easybuild/easyconfigs/p/PRRTE/PRRTE-3.0.5-GCCcore-13.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    ('PMIx', '5.0.2'),
+]
+
+configopts = ' --with-libevent=$EBROOTLIBEVENT'
+configopts += ' --with-hwloc=$EBROOTHWLOC --with-pmix=$EBROOTPMIX'
+
+buildopts = 'V=1'
+
+sanity_check_paths = {
+    'files': ['bin/prte', 'bin/prte_info', 'bin/prterun', 'bin/prun', 'bin/pterm'],
```
Comment (bedroge) [inline easybuild/easyconfigs/p/PRRTE/PRRTE-3.0.5-GCCcore-13.3.0.eb:30]: @bartoldeman Could you add sanity check commands for these binaries?

**Comment by bartoldeman** [comment]
Test report by @bartoldeman
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3347
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
login1.int.archimedes.c3.ca - Linux Rocky Linux 8.9 (Green Obsidian), x86_64, Intel Xeon Processor (Skylake, IBRS) (cascadelake), Python 3.11.4
See https://gist.github.com/bartoldeman/4257011ae2adb6e7e26f6bfd1735c987 for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3347
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
interactive1 - Linux Rocky Linux 8.9, x86_64, AMD EPYC-Milan Processor (zen2), Python 3.6.8
See https://gist.github.com/bedroge/480dd6c59bb0f8af85da7fb5b01ab6f8 for a full test report.

File: easybuild/easyconfigs/p/PRRTE/PRRTE-3.0.5-GCCcore-13.3.0.eb (hunk 1 above)
Comment (bartoldeman) [reply easybuild/easyconfigs/p/PRRTE/PRRTE-3.0.5-GCCcore-13.3.0.eb]: Done

**Comment by bedroge** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--include-easyblocks-from-pr 3347"

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3347
**SUCCESS**
Build succeeded for 1 out of 1 (2 easyconfigs in total)
bear-pg0207u28a.bear.cluster - Linux RHEL 8.6, x86_64, AMD EPYC 9554 64-Core Processor (zen4), Python 3.6.8
See https://gist.github.com/branfosj/9ac0fc8505ef00527c8734b2316fc9b2 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3347
**SUCCESS**
Build succeeded for 1 out of 1 (2 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/bc102892f3e6db53afd730608a5c559a for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3347
**SUCCESS**
Build succeeded for 1 out of 1 (2 easyconfigs in total)
bear-pg0207u20a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8480CL (sapphirerapids), Python 3.6.8
See https://gist.github.com/branfosj/9a869ac197b1667e92d0ee75fceeb3b7 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3347
**SUCCESS**
Build succeeded for 1 out of 1 (2 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/branfosj/7dc2f78f926c8681c26b34e054a4c248 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3347
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0207u28a.bear.cluster - Linux RHEL 8.6, x86_64, AMD EPYC 9554 64-Core Processor (zen4), Python 3.6.8
See https://gist.github.com/branfosj/49fd8e2092a1706458e6d278907ad36b for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3347
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/branfosj/7090c76de4bdff7a29aa26928be12fed for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3347
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/a96b6951a26ced83fdd78ec3838e6cab for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3347
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0207u20a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8480CL (sapphirerapids), Python 3.6.8
See https://gist.github.com/branfosj/31805e8168109083fcd1dfcbd7161db0 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @bartoldeman!

**Pull request merged**